contracts/error/Errors.sol

```diff
@@ -198,6 +199,9 @@ library Errors {
     error InvalidMarketTokenBalanceForClaimableFunding(address market, address token, uint256 balance, uint256 claimableFundingFeeAmount);
     error UnexpectedPoolValue(int256 poolValue);
 
+    // MarketPositionImpactUtils errors
+    error InvalidImpactPoolValueForWithdrawal(uint256 poolValue);
```

Comment:
could change this to InsufficinetImpactPoolValueForWithdrawal(withdrawalAmount, poolValue)